shared/actions/teams.js

```diff
@@ -482,124 +483,129 @@ function* getDetails(_, action) {
   const {teamname} = action.payload
   yield Saga.put(TeamsGen.createGetTeamOperations({teamname}))
   yield Saga.put(TeamsGen.createGetTeamPublicity({teamname}))
-  const unsafeDetails: RPCTypes.TeamDetails = yield* Saga.callPromise(
-    RPCTypes.teamsTeamGetRpcPromise,
-    {
-      name: teamname,
-    },
-    Constants.teamWaitingKey(teamname)
-  )
-
-  // Don't allow the none default
-  const details: RPCTypes.TeamDetails = {
-    ...unsafeDetails,
-    settings: {
-      ...unsafeDetails.settings,
-      joinAs:
-        unsafeDetails.settings.joinAs === RPCTypes.teamsTeamRole.none
-          ? RPCTypes.teamsTeamRole.reader
-          : unsafeDetails.settings.joinAs,
-    },
-  }
 
-  // Get requests to join
-  let requests
-  const state = yield* Saga.selectState()
-  if (Constants.getCanPerform(state, teamname).manageMembers) {
-    // TODO (DESKTOP-6478) move this somewhere else
-    requests = yield* Saga.callPromise(
-      RPCTypes.teamsTeamListRequestsRpcPromise,
+  try {
```

Comment:
This is now wrapped in a try block to avoid the black box. The error is still visible in the show more option of the reloadable component.